python/ParkingLot.py

```diff
@@ -31,15 +31,25 @@ def countSensors(self):
     
     
     # Returns the difference of totalSpots in parking lot and available spots
+    # def countAvailableSpots(self):
+    #     spotsTaken = 0
+    #     col = list(self.collection.find())
+
+    #     for document in col:
+    #         if document["isVacant"] == False:
+    #             spotsTaken += 1
+        
+    #     return self.totalSpots - spotsTaken
+
     def countAvailableSpots(self):
-        spotsTaken = 0
-        col = list(self.collection.find())
+        availableSpots = 0
```

Comment:
Is it possible to have const variables in python? Otherwise i have no issues

Reply:
ok. i believe our only constant variables so far are the total number of spots for each parking lot.

Reply:
Yes, there's a way to have constant variables in Python. You have to import a module called "Constant". What do you want to make constant?